openapi_python_client/utils.py

```diff
@@ -1,12 +1,40 @@
+import keyword
 import re
 
 import stringcase
 
 
+def _shadow_keyword(value: str) -> str:
+    """If the value is a keyword, append an underscore
+    """
+    if keyword.iskeyword(value):
+        return f"{value}_"
+    return value
+
+
+def sanitize(value: str) -> str:
+    """Sanitize value to be a valid python identifier
+
+    Uses the following grammar:
+    identifier ::=  (letter|"_") (letter | digit | "_")*
+    letter     ::=  lowercase | uppercase
+    lowercase  ::=  "a"..."z"
+    uppercase  ::=  "A"..."Z"
+    digit      ::=  "0"..."9"
+    """
+    if value.isidentifier():
+        return _shadow_keyword(value)
+    # Drop invalid characters
+    value = re.sub("[^0-9a-zA-Z_]", "", value)
+    # Drop invalid start characters
+    value = re.sub("^[^a-zA-Z_]+", "", value)
+    return _shadow_keyword(value)
+
+
 def snake_case(value: str) -> str:
     value = re.sub(r"([A-Z]{2,})([A-Z][a-z]|[ -_]|$)", lambda m: m.group(1).title() + m.group(2), value.strip())
```

Comment:
I think `[ -_]` might be unintentionally including a bunch of characters (ASCII 32 - 95).
I'm not entirely sure of the intent behind these two regexes.